src/monthly-timeseries/MonthlyTimeseries.js

```diff
@@ -0,0 +1,82 @@
+import PropTypes from "prop-types";
+import React from "react";
+import ResponsiveOrdinalFrame from "semiotic/lib/ResponsiveOrdinalFrame";
+import styled from "styled-components";
+import { THEME } from "../constants";
+import { formatAsPct } from "../utils";
+
+const MonthlyTimeseriesWrapper = styled.div`
+  height: 350px;
+  width: 100%;
+
+  /* classes provided by Semiotic */
+  .frame {
+    .axis-baseline {
+      stroke: ${(props) => props.theme.colors.chartAxis};
+    }
+
+    .axis-label,
+    .ordinal-labels {
+      fill: ${(props) => props.theme.colors.chartAxis};
+      font: ${(props) => props.theme.fonts.body};
+      font-size: 12px;
+    }
+
+    .background-graphics,
+    .visualization-layer {
+      shape-rendering: crispEdges;
+    }
+  }
+`;
+
+const chartMargins = {
+  top: 10,
+  bottom: 20,
+  left: 50,
+  right: 0,
+};
+
+const TimeLabel = styled.text`
+  text-anchor: middle;
+`;
+
+export default function MonthlyTimeseries({ data }) {
```

Comment:
Is dealing with Monthly data our only use case?

Reply:
for now, yes!